specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2019-10-01/attestations.json

```diff
@@ -0,0 +1,779 @@
+          "Get attestation at individual resource scope": {
+            "$ref": "./examples/Attestations_GetResourceScope.json"
+          }
+        }
+      },
+      "delete": {
+        "operationId": "Attestations_DeleteAtResource",
+        "description": "Deletes an existing attestation at individual resource scope.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceIdParameter"
+          },
+          {
+            "$ref": "#/parameters/attestationNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The attestation was successfully deleted."
+          },
+          "204": {
+            "description": "The attestation did not exist."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete attestation at individual resource scope": {
+            "$ref": "./examples/Attestations_DeleteResourceScope.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AttestationListResult": {
+      "description": "List of attestations.",
+      "properties": {
+        "value": {
+          "description": "Array of attestation definitions.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Attestation"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of results.",
+          "readOnly": true
+        }
+      }
+    },
+    "Attestation": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/AttestationProperties",
+          "description": "Properties for the attestation."
+        },
+        "systemData": {
+          "readOnly": true,
+          "type": "object",
+          "description": "Azure Resource Manager metadata containing createdBy and modifiedBy information.",
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/systemData"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/Resource"
+        }
+      ],
+      "required": [
+        "properties"
+      ],
+      "description": "An attestation resource."
+    },
+    "AttestationProperties": {
+      "properties": {
+        "policyAssignmentId": {
+          "type": "string",
+          "description": "The resource ID of the policy assignment that the attestation is setting the state for."
+        },
+        "policyDefinitionReferenceId": {
```

Comment:
Can't we get this using policyAssignment?
What will happen to these objects when policy definition or assignment gets removed?

Reply:
This is when the assignment is pointing to a policySetDefinition.  policySetDefinitions contain references to many different policyDefinitions, each one is identified by a policyDefinitionReferenceId.  I'll improve the description